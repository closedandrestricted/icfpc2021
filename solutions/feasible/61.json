{"vertices":[[39,33],[24,13],[11,40],[7,32],[1,20],[30,22],[29,23],[28,28],[10,46],[49,43],[37,54],[26,16],[12,43],[11,46],[42,42]]}