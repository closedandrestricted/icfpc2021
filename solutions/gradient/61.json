{"vertices":[[69,86],[48,100],[45,70],[11,37],[40,77],[33,25],[33,48],[30,104],[62,59],[58,37],[40,78],[97,64],[95,33],[127,79],[122,47]]}