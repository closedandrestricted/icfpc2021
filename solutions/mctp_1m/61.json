{"vertices":[[59,75],[38,61],[63,45],[67,42],[56,78],[60,19],[59,20],[36,97],[60,50],[85,31],[38,68],[90,68],[116,52],[118,50],[117,82]]}