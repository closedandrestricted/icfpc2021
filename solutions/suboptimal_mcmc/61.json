{"vertices":[[58,98],[45,76],[28,102],[18,75],[37,52],[37,60],[42,81],[20,29],[64,59],[64,63],[48,35],[97,70],[93,39],[128,83],[121,51]]}