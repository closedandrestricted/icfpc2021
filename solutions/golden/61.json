{"vertices":[[31,100],[56,100],[43,72],[12,42],[68,79],[34,30],[36,48],[68,50],[67,50],[61,38],[66,21],[98,66],[99,36],[126,83],[125,51]]}